src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -72,7 +72,7 @@ public void MercuryLoginTest_006_closeButtonWorksAndRedirectsProperly() {
     Assertion.assertEquals(expectedHomePageTitle, homePage.getArticleTitle());
   }
 
-  @Test(groups = "MercuryLoginTest_007")
+  @Test(groups = {"MercuryLoginTest_007", "LoginTests_002"})
   public void MercuryLoginTest_007_registerNowLinkWorks() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)